src/app.js

```diff
@@ -1,17 +1,33 @@
 import express from 'express'
 import userRouter from './router/userRouter'
+import internalRouter from './router/internalRouter'
 import handleValidationError from './error/handleValidationError'
 import handleError from './error/errorHandler'
+import logMethodInvocation from './logger/infoLoggerMidelware'
+import { generalLogger as logger } from './logger/logger'
 
 const PORT = process.env.PORT || 8182
 const app = express()
 
 app.use(express.json())
 app.use(express.urlencoded({ extended: false }))
 
+app.use(logMethodInvocation)
```

Comment:
![image](https://user-images.githubusercontent.com/52933810/146561732-e6a4bea3-5433-473c-98c4-af14724aaff7.png)

As I understand from the task requiremnets it shouldn't be a global middleware which will measure time for all requests, instead it should be a wrapper or decorator for a controller which you can use wherever you need, smth like this:

```
const measureExecTime = (controller) => {
  return (req, res, next) => {
    const start = process.hrtime.bigint();

    res.on("finish", () => {
      const end = process.hrtime.bigint();

      // calculate exec time and log it
    });

    controller(req, res, next);
  };
};
```

You can use it in `userRouter.js` to wrap your controllers: 

```
router.route('/')
  .get(getAllUsers)
  .post(userModelValidator, measureExecTime(createUser))
```